x-pack/plugins/observability_solution/slo/public/pages/slos/components/card_view/slo_card_item.tsx

```diff
@@ -108,6 +108,7 @@ export function SloCardItem({ slo, rules, activeAlerts, historicalSummary, refet
   return (
     <>
       <EuiPanel
+        className="sloCardItem"
```

Comment:
I guess this `className` is just for the test, why don't use `data-test-subj` instead?